### ISSUE TITLE
Nice to have worker task:topic mapping in Sink connector

### ISSUE THREAD

**Issue by jcocchi** [microsoft/kafka-connect-cosmosdb#83]
Currently the sink connector spins up as many workers as the maxTasks configuration and assigns all topics to all workers. Because of the topic -> collection mapping it would be more efficient to assign specific works to a subset of topics instead of all having all workers read from all topics.

Investigate the implications of this and implement the best solution.

This refers to the [taskConfigs function](https://github.com/microsoft/kafka-connect-cosmosdb/blob/master/src/main/scala/com/microsoft/azure/cosmosdb/kafka/connect/sink/CosmosDBSinkConnector.scala#L46) of the CosmosDBSinkConnector

**Comment by brandynbrown** [comment]
Follow up: do a spike to see if it's feasible to configure a worker to a specific topic

If so, 
Are there performance improvements?

**Comment by brandynbrown** [comment]
Spike
Investigate what Cassandra connector does when monitoring multiple topics.

**Comment by brandynbrown** [comment]
blocked by #292
